### DISCUSSION TITLE
workflow instance execute endpoint with extra dynamic payload props

### DISCUSSION THREAD

**Discussion by ricksearcy** [elsa-workflows/elsa-core#1643] [Q&A]
When using the execute endpoint how can we send in additional data beyond the correlationId? I can use the correlationId js property and set it to a property of the 1st activity fine, but I need additional dynamic props not set statically in the designer. 

Example:
POST
.../workflows/b35b1a5b06794e57a625bdf027696143/execute
{
"correlationId":"a732c381-0add-4ad8-8325-9cf9caa2d837",
"myProp":"myValue"
}

**Comment by sfmskywalker** [comment]
You were very close to the solution. You can send extra props like this:

```http
POST
.../workflows/b35b1a5b06794e57a625bdf027696143/execute
{
"correlationId":"a732c381-0add-4ad8-8325-9cf9caa2d837",
"input": {
   "myProp":"myValue"
}
}
```
